frontend/src/js/prime-web-receiver-pre.js

```diff
@@ -225,53 +275,37 @@ function processJwtToken(){
     let claims = token ? JSON.parse(atob(token.split('.')[1])) : null;
 
     if (token && claims && moment().isBefore(moment.unix(claims.exp))) {
-
+        // update the email user link
         const emailUser = document.getElementById("emailUser");
         if (emailUser) emailUser.innerHTML = claims.sub;
+        // refresh the logout link
         const logout = document.getElementById("logout");
         if (logout) logout.innerHTML = 'Logout';
-
+        // refresh our signin link/remove it
         const _signIn = document.getElementById("signInButton");
         if (_signIn) { _signIn.setAttribute( "hidden", "hidden"); }
-
+        // process the org so the dropdown looks correct
         const _org = claims.organization.filter(c => c !== "DHPrimeAdmins");
-
         const oldOrg = window.sessionStorage.getItem( "oldOrg");
-
-        window.org = oldOrg? oldOrg : (_org && _org.length > 0) ? _org[0] : null;
-
+        window.org = oldOrg ? oldOrg : (_org && _org.length > 0) ? _org[0] : null;
         window.sessionStorage.setItem( "oldOrg", window.org );
-        window.orgs = _org;
         window.user = claims.sub;
+        // set the token here
         window.jwt = token;
 
-        if (claims.organization.includes( "DHPrimeAdmins" ) ) {
-            const dropDownWrapper = document.getElementById("orgDropdown");
-            dropDownWrapper.classList.remove("display-none");
-            dropDownWrapper.classList.add("display-block");
-            const _dropdown = document.createElement("div");
-            _dropdown.id = "dropdown";
-
-            let _orgsOptions = "";
-
-            window.orgs.forEach( org => {
-                _orgsOptions +=
-                    `
-                        <option value="${org}" ${window.org === org ? 'selected="selected"' : ""} >
-                            ${convertOrgName(org).toUpperCase()}
-                        </option>
-                    `;
+        if (claims.organization.includes("DHPrimeAdmins")) {
```

Comment:
Consider making `DHPrimeAdmins` a constant; it's used here and on L305, L288 and probably some other places too